src/bundles/SellerRegistration/components/DocumentsForm/DocumentsForm.js

```diff
@@ -270,7 +270,8 @@ class DocumentsForm extends BaseForm {
                                             <p styleName="filler">&nbsp;</p>
                                             <p id={`span_${key}`} ref={`span_${key}`} styleName="file-name">{!isEmpty(name) && name}</p>
                                             <button type="submit" styleName="submit-container" onClick={this.onUpload.bind(this, key)}>Upload</button>
-                                            <p styleName="custom-input">Choose file</p>
+                                            <input type="file" id={key} name={key} styleName="hidden-input" accept=".pdf,.jpg,.png" onChange={this.onChange.bind(this, key)} />
```

Comment:
are these all the formats we are accepting?

Reply:
Yes - in the sign up flow it says those are the 3 file formats the marketplace were accepting.